openedx/adg/lms/our_team/admin.py

```diff
@@ -0,0 +1,18 @@
+"""
+Registering models for our team app.
+"""
+from django.contrib import admin
+
+from .models import OurTeamMember
+
+
+@admin.register(OurTeamMember)
+class ApplicationHubAdmin(admin.ModelAdmin):
+    """
+    Django admin class for OurTeam
+    """
+
+    fields = ('name', 'designation', 'image', 'description', 'url',)
```

Comment:
No need for this. By default all the fields will be displayed at the change/update view.